tests/unit/App.spec.ts

```diff
@@ -1001,6 +940,15 @@ describe('Restoration - App page', () => {
 })
 
 describe('Breadcrumbs for firms - Without Easy Legal Name Fix', () => {
+  beforeAll(() => {
+    // mock parties
+    vi.spyOn((LegalServices as any), 'fetchParties').mockResolvedValue({
+      parties: [...filingData.incorporationApplication.parties]
+    })
```

Comment:
'fetchParties' is unique to firms (ie, this test suite), so I mock it (while the test suites above didn't need this).